frontend/projects/ui/src/app/pages/server-routes/sessions/sessions.page.html

```diff
@@ -8,58 +8,95 @@
+            style="width: 40px; height: 40px; border-radius: 0"
+          ></ion-skeleton-text>
         </ion-avatar>
         <ion-label>
-          <ion-skeleton-text animated style="width: 150px; height: 20px; margin-bottom: 10px;"></ion-skeleton-text>
-          <ion-skeleton-text animated style="width: 250px; height: 14px; margin-bottom: 12px;"></ion-skeleton-text>
-          <ion-skeleton-text animated style="width: 350px;"></ion-skeleton-text>
+          <ion-skeleton-text
+            animated
+            style="width: 150px; height: 20px; margin-bottom: 10px"
+          ></ion-skeleton-text>
+          <ion-skeleton-text
+            animated
+            style="width: 250px; height: 14px; margin-bottom: 12px"
+          ></ion-skeleton-text>
+          <ion-skeleton-text animated style="width: 350px"></ion-skeleton-text>
         </ion-label>
         <ion-button *ngIf="entry === 'second'" slot="end" fill="clear">
-          <ion-skeleton-text animated style="width: 60px; border-radius: 0"></ion-skeleton-text>
+          <ion-skeleton-text
+            animated
+            style="width: 60px; border-radius: 0"
+          ></ion-skeleton-text>
         </ion-button>
       </ion-item>
     </div>
   </ion-item-group>
 
   <!-- not loading -->
   <ion-item-group *ngIf="!loading">
-
-    <ion-item-divider>This Session</ion-item-divider>
-    <ion-item *ngIf="sessionInfo.sessions[sessionInfo.current] as current">
-      <ion-icon slot="start" size="large" [name]="getPlatformIcon(current.metadata.platforms)"></ion-icon>
+    <ion-item-divider>Current Session</ion-item-divider>
+    <ion-item>
+      <ion-icon
+        slot="start"
+        size="large"
+        [name]="getPlatformIcon(currentSession.metadata.platforms)"
+      ></ion-icon>
       <ion-label>
-        <h1>{{ getPlatformName(current.metadata.platforms) }}</h1>
-        <h2>Last Active: {{ current['last-active'] | date : 'medium' }}</h2>
-        <p>{{ current['user-agent'] }}</p>
+        <h1>{{ getPlatformName(currentSession.metadata.platforms) }}</h1>
+        <h2>
+          Last Active: {{ currentSession['last-active'] | date : 'medium' }}
+        </h2>
+        <p>{{ currentSession['user-agent'] }}</p>
       </ion-label>
     </ion-item>
 
-    <ion-item-divider>Other Sessions</ion-item-divider>
-    <div *ngFor="let session of sessionInfo.sessions | keyvalue : asIsOrder;">
-      <ion-item 
-        [id]="session.key"
-        *ngIf="session.key !== sessionInfo.current"
+    <ion-item-divider>
+      Other Sessions
+      <ion-button
+        *ngIf="!(otherSessions | empty)"
+        slot="end"
+        fill="clear"
+        color="danger"
+        strong
+        (click)="presentAlertKillAll()"
       >
-        <ion-icon slot="start" size="large" [name]="getPlatformIcon(session.value.metadata.platforms)"></ion-icon>
+        Kill All
+      </ion-button>
+    </ion-item-divider>
+    <div *ngFor="let session of otherSessions">
+      <ion-item [id]="session.id">
```

Comment:
Don't think we need `[id]="session.id"`.

Try removing and see if it throws an error on page load with at least 2 entries in the otherSessions list.

Reply:
It does not throw an error in the outlined case - removing.